src/cpu/ppc64/PPC64_gemm_s8x8s32.cpp

```diff
@@ -0,0 +1,186 @@
+#ifdef __MMA__
+#include <altivec.h>
+#include "PPC64_gemm_s8x8s32.hpp"
+#include "cpu/simple_q10n.hpp"
+//#define TIMING 1
+
+#ifdef TIMING
```

Comment:
Please move all timing under verbose functionality. If it requires a special timer on PPC64, it would be fine.
Just in case, will std::chrono work?

Reply:
I can just remove the timing stuff entirely.  That was my plan, I just forgot it...

Reply:
fixed in fourth commit